src/test/java/com/rocketden/main/service/SubmitServiceTests.java

```diff
@@ -0,0 +1,65 @@
+package com.rocketden.main.service;
+
+import com.rocketden.main.dto.game.GameMapper;
+import com.rocketden.main.dto.game.SubmissionRequest;
+import com.rocketden.main.dto.user.UserMapper;
+import com.rocketden.main.game_object.Game;
+import com.rocketden.main.game_object.Submission;
+import com.rocketden.main.model.Room;
+import com.rocketden.main.model.User;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+public class SubmitServiceTests {
+
+    private static final String NICKNAME = "rocket";
+    private static final String NICKNAME_2 = "rocketrocket";
+    private static final String ROOM_ID = "012345";
+    private static final String USER_ID = "098765";
+    private static final String CODE = "print('hi')";
+    private static final String LANGUAGE = "python";
+
+    @Spy
+    @InjectMocks
+    private SubmitService submitService;
+
+    @Test
+    public void submitSolutionSuccess() {
+        // TODO: For now, this just checks that the submission is stored correctly.
+        // In the future, it will likely call the tester endpoint and send a socket update too.
```

Comment:
```suggestion
        /**
         * TODO: For now, this just checks that the submission is stored correctly.
         * In the future, it will likely call the tester endpoint and send a socket update too.
         */
```